tsl/src/compression/compression.h

```diff
@@ -260,6 +261,7 @@ typedef struct RowCompressor
 	int64 num_compressed_rows;
 	/* if recompressing segmentwise, we use this info to reset the sequence number */
 	bool reset_sequence;
+	bool update_sequence;
```

Comment:
minor, might be useful to add a comment explaining when this field is used?

Reply:
Fixed, added comment.